### PR TITLE
[relates #281] DB verifying steps to scenarios. part 2

### PR DESCRIPTION
**Pull request by Stefan365**
part  2. + random corrections

### CODE REVIEW

File: ui-tests/src/test/resources/features/utils/debug-step.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@debug-step
```
Comment (dsimansk) [inline ui-tests/src/test/resources/features/utils/debug-step.feature]: Should this be included in PR also?

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java [hunk 2]
```diff
@@ -124,14 +125,20 @@ public void insertsIntoTable(String tableName, DataTable data) throws SQLExcepti
...
 		Thread.sleep(ms + 1000);
 
 		ResultSet rs;
 		List<Integer> completedAll = new ArrayList<>();
-		String sql = String.format("SELECT completed FROM todo WHERE task like %s", task);
+		String sql = String.format("SELECT completed FROM todo WHERE task like '%s'", task);
+		log.info("SQL **{}**", sql);
 		rs = dbUtils.readSqlOnSampleDb(sql);
-		Assertions.assertThat(rs).isEqualToComparingOnlyGivenFields(val, "completed");
+//		does not work with resultset:
```
Comment (dsimansk) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java]: Yes, we would need `assertj-db` for that. This solution is all right. You can get rid off the commented code.

File: ui-tests/src/test/resources/features/integrations/database-to-database-with-extension.feature [hunk 3]
```diff
@@ -21,7 +21,7 @@ Feature: Upload tech extension and add it to integration
     When Camilla clicks on the "Import Extension" button
     Then she is presented with the Syndesis page "Import Extension"
 
-    When Camilla upload extension
+    When Camilla upload extension "extension_name"
```
Comment (dsimansk) [inline ui-tests/src/test/resources/features/integrations/database-to-database-with-extension.feature]: The name should be ` When Camilla upload extension "syndesis-extensions-1.0.0-SNAPSHOT"`

File: ui-tests/src/test/resources/features/utils/debug-step.feature (hunk 1 above)
Comment (Stefan365) [reply ui-tests/src/test/resources/features/utils/debug-step.feature]: Imenat so. perhaps it could be useful too also for others.

Comment (dsimansk) [reply ui-tests/src/test/resources/features/utils/debug-step.feature]: Ok, maybe rename the file to `debug-step.feature.example` or `debug-step.example` something like that to be excluded from execution with whole testsuite.

Comment (Stefan365) [reply ui-tests/src/test/resources/features/utils/debug-step.feature]: done

File: ui-tests/src/test/resources/features/integrations/salesforce-to-database.feature [hunk 4]
```diff
@@ -70,11 +73,12 @@ Feature: Test to verify correct function of connections kebab menu
     And she defines integration name "Salesforce to PostresDB E2E"
     And clicks on the "Publish" button
       # assert integration is present in list
-    Then Camilla is presented with "Salesforce to PostresDB E2E" integration details
+#    Then Camilla is presented with "Salesforce to PostresDB E2E" integration details
```
Comment (dsimansk) [inline ui-tests/src/test/resources/features/integrations/salesforce-to-database.feature]: We don't wan to check the detail page anymore?

**Comment by Stefan365** [comment]
done

**Comment by Stefan365** [comment]
done

**Pull request merged**